docs/OnlineTutorial/guide.md

```diff
@@ -1484,6 +1484,9 @@ it must be allocated with the `new` keyword, but for now
 we will only work with methods that take a previously allocated array as an
 argument. (See the tutorial on memory for more on allocation.)
 
+If a variable's value might be null or an array, then its type must be
+`array?<...>` instead of `array<...>` (which has only non-null values).
```

Comment:
Great. Can we reconcile this with L1475f?

Reply:
Revised